### PR TITLE
Add Bazelisk support to Linux and Windows

### PR DESCRIPTION
**Pull request by nikita-bykov**
PR for Issue: #221

This adds installation Bazelisk for Windows 2016,  Windows 2019, Ubuntu 16.04, Ubuntu 18.04 using `npm`

### CODE REVIEW

File: images/win/scripts/Installers/Validate-Bazel.ps1 [hunk 1]
```diff
@@ -13,11 +13,30 @@ else
     exit 1
```
Comment (al-cheb) [inline images/win/scripts/Installers/Validate-Bazel.ps1]: Are you sure what we shouldn't include bazel to the path?

**Comment by al-cheb** [comment]
nitpick: maybe we should cut bazel version output:

```
Bazel (Build label: 2.1.1
Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Fri Feb 21 14:56:47 2020 (1582297007)
Build timestamp: 1582297007
Build timestamp as int: 1582297007)
```

**Comment by maxim-lobanov** [comment]
> nitpick: maybe we should cut bazel version output:
> 
> ```
> Bazel (Build label: 2.1.1
> Build target: bazel-out/k8-opt/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
> Build time: Fri Feb 21 14:56:47 2020 (1582297007)
> Build timestamp: 1582297007
> Build timestamp as int: 1582297007)
> ```

@al-cheb , we definitely should.
@nikita-bykov ^

**Pull request merged**